app/helpers/projects_helper.rb

```diff
@@ -4,4 +4,19 @@ def project_row_class(project)
 
     project_class = 'border-2 border-orange-300 bg-orange-100' if project.highlight
   end
+
+  def format_country(country)
```

Comment:
Created helpers in project as it's specific to project for now; if there's a preference for elsewhere, happy to change.